test/unit/san.c

```diff
@@ -114,6 +114,7 @@ verify_pmuzzy(unsigned arena_ind, uint64_t expected) {
 }
 
 TEST_BEGIN(test_guarded_decay) {
+	test_skip_if(config_limit_usize_gap);
```

Comment:
For a core change like this, we should make the test more aware of the usize and not skipping it. Since we plan to change to default on soon, we won't want these tests to be completely skipped in the future.

Reply:
Yes, this is a temporary mitigation: it failed on Cirrus but not locally. It seems somehow `san_guard_large` is viewed as an invalid conf there. Looking into it now. Similar for `retained.c`, I am looking for a better way to get more accurate pdirty assertion with the change.